srcpkgs/gnome-session/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'gnome-session'
 pkgname=gnome-session
-version=3.34.1
+version=3.36.0
 revision=1
 build_style=meson
 configure_args="-Dsystemd_journal=false"
```

Comment:
Needs `-Dsystemd_session=disable` as well.

Reply:
Added.

Reply:
`gnome-session` no longer recognizes `--systemd` option, so I couldn't login to gnome-flashback. I've removed the `--systemd` flag in `/usr/libexec/gnome-flashback-{compiz,metacity}`.